### DISCUSSION TITLE
How to make Call tree visible on full screen

### DISCUSSION THREAD

**Discussion by vpulkit007** [certinia/debug-log-analyzer#37] [Q&A] [answered]
Hi Team,

Can anyone tell me is there any feature to make callTree visible on full screen? 
This will save a lot of time 

![Uploading Screenshot (13).png…]()

**Comment by vpulkit007** [comment]
![Screenshot (13)](https://user-images.githubusercontent.com/57859702/128178157-661e3769-e824-4c98-abbc-dc5330df3c16.png)

**Reply by vpulkit007** [reply]
Image reference

**Reply by kjonescertinia** [reply]
Oh, that is pretty ugly. I think the easy thing we can do is trim that text so it does not take up the whole screen. I will see if we can add a way to minimise it as well.

**Reply by vpulkit007** [reply]
How to trim the text?

**Reply by nawforce** [reply]
We will add something in the code to do that automatically, for now though I would suggest you edit your log file to shorten those messages and then you find it easier to use the tool.

**Reply by lcottercertinia** [reply]
@vpulkit007 
We have just closed some issues which should resolve this #47 and #53.
There are some screen shots of the new behaviour if you wanted a sneak peek 👍🏻

**Comment by lcottercertinia** [comment, marked as answer]
This should be resolved by issues #47 and #53
